wgpu-core/src/command/allocator.rs

```diff
@@ -27,10 +27,10 @@ impl<B: hal::Backend> CommandPool<B> {
     fn maintain(&mut self, lowest_active_index: SubmissionIndex) {
         for i in (0..self.pending.len()).rev() {
             if self.pending[i].1 < lowest_active_index {
-                let cmd_buf = self.pending.swap_remove(i).0;
+                let (cmd_buf, index) = self.pending.swap_remove(i);
```

Comment:
I don't understand how this could be `None` if we just used `self.pending[i].1` in the comparison above. Do you?

Reply:
nvm, I see now (facepalm)